index.html

```diff
@@ -0,0 +1,51 @@
+<html lang="ja">
+  <head>
+    <meta charset="utf-8">
+    <title>1-3</title>
+  </head>
+  <body>
+    
+    <script>
+      'use strict';
+      
+      const items = ['掃除',"買い物","散歩"];
+      const title = ('======================= \n 現在持っているタスク一覧 \n=======================');
+      console.log(title);
+      
+      const itemMap = function(){
```

Comment:
```suggestion
      const displayItems = function(){
```
メソッドや関数の名前は「何をやってくれるのか」がわかるような名前をつけるように心がけてください。
具体的には「名詞のみ」ではなく「自動詞」または「他動詞＋目的語」となるようにすると考えやすいです。